scripts/lint.sh

```diff
@@ -55,6 +55,15 @@ format(){
 }
 
 lint(){
+    # === Spell check
+    if ! command -v cspell >/dev/null 2>&1; then
+        printf "%s\n" "cspell is not installed. Skipping spellcheck"
+    else
+    # NOTE: Keep this command in sync with `.github/workflows/build.yml`
+        cspell -c cspell.config.yaml --dictionary cspell-custom-words.txt "**.*md"
```

Comment:
the config contains the dictionary, and [I'm pretty sure](https://cspell.org/docs/getting-started#1-create-a-configuration-file) it will automatically find that config.